src/main/java/uk/gov/hmcts/auth/provider/service/api/auth/AuthController.java

```diff
@@ -26,6 +29,11 @@ public AuthController(AuthService authService) {
     }
 
     @PostMapping("/lease")
+    @ApiOperation("Sign in, lease auth token")
+    @ApiResponses({
+        @ApiResponse(code = 200, message = "JWT token"),
+        @ApiResponse(code = 401, message = "Invalid service & otp pair")
```

Comment:
Is this the only case? Maybe it could be more general, i.e. `Unauthorised. Returns error details.` or sth?